### PR TITLE
Some getting started doc for AWS

### PR DESCRIPTION
**Pull request by dpoirier**

### CODE REVIEW

**Comment by dpoirier** [comment]
This pull request is old but I think still valid - maybe @tobiasmcnulty and/or @copelco would like to review?

File: source/aws.rst [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+an IAM user within that AWS account, but a user in the global Amazon system of
+users - the same set of users that can buy things on Amazon's shopping
+sites, in fact. To create a new AWS account, you have to log in as one of
+this type of user.
+
+Logging in
+----------
+
+To login using a *root user*, there's a global url:
+`https://www.amazon.com/ap/signin <https://www.amazon.com/ap/signin>`_.
```
Comment (vkurup) [inline source/aws.rst]: This URL now 404s... Unfortunately, I don't there is a direct link that is guaranteed to work. The AWS docs suggest going to the main console login and then clicking on the 'root url' link if you're not at the right place: https://docs.aws.amazon.com/IAM/latest/UserGuide/console.html#BrowserDefault

:sigh:

Comment (dpoirier) [reply source/aws.rst]: Thanks for spotting that. I rewrote that part to just go to the main signin and use the 'root url' link.

I also added a section about how to get in and out of subaccounts, which I don't think we'd been doing yet when I wrote this. Please see if that makes sense.  Also, do you think I need to say anything about creating subaccounts or giving people access to them?  (I'd have to find out myself.)

Comment (vkurup) [reply source/aws.rst]: Nice! Thanks for adding that info about subaccounts. I've also never created a subaccount, so not sure it needs to be documented here.  :shipit:

**Pull request merged**